src/css/components/_text-track.scss

```diff
@@ -24,3 +24,7 @@
 .vjs-subtitles { color: #fff /* Subtitles are white */; }
 .vjs-captions { color: #fc6 /* Captions are yellow */; }
 .vjs-tt-cue { display: block; }
+
+video::-webkit-media-text-track-display {
+  @include transform(translateY(-45px));
```

Comment:
This should probably be `4em` rather than `45px`. `4.5em` is a bit much, I think.
Also, can we add this value as a variable that defaults to `4em` to the [vars](https://github.com/videojs/video.js/blob/master/src/css/_variables.scss) file? That way, a user could more easily change where the captions are if they have a custom control bar and are including the sass files.